woocommerce/woocommerce-ios
# Shipping Labels: Render package item for new package details screen (supporting multiple packages)

Part of #4599 
This PR depends on #4959 so please make sure that PR is reviewed and merged first.

# Description
This PR updates package item UI and view model to display rows of order items.

# Changes
- Updates package item view model to configure item rows based on products, product variations and order items.
- Updates package item UI to render rows for order items and package details section. Selection of package doesn't work yet and will be implemented in the next PR.
- Adds unit tests to check order items of package item.

# Screenshot
<img src="https://user-images.githubusercontent.com/5533851/132660843-51a90df6-adee-4053-93eb-e77ac2ed285c.png" width=400 />

# Testing
1. Make sure that your test store has installed WCShip plugin and configured packages in WPAdmin > WooCommerce > Settings > Shipping > WooCommerce Shipping.
2. Open an order detail that is eligible for the shipping label.
3. Configure Ship To and Ship From.
4. Tap Continue on Package Details row. Notice that you're navigated to a new Package Details screen with title, order item rows and package details section. UI should look as before, but no validation / package selection is implemented yet.

Update release notes:
